source/configuration_manual/fts/squat.rst

```diff
@@ -3,11 +3,9 @@
 Squat FTS Engine
 ================
 
-.. deprecated:: v2.1
+.. versionremoved:: 2.4.0;3.0.0
 
-.. warning:: Squat is deprecated and should no longer be used.
-             It is going to be removed in a future version entirely.
-	     It's recommended to use :ref:`fts_backend_solr` instead.
+.. warning:: It's recommended to use :ref:`fts_backend_solr` instead.
```

Comment:
I  think we could just remove this. People can choose themselves some other fts engine.